gradle/libs.versions.toml

```diff
@@ -83,6 +83,17 @@ coroutines = "1.4.2"
 # Local Server
 ktor = "1.6.0"
 
+# Otel
+slf4j = "1.7.30"
+moshi = "1.11.0"
```

Comment:
moshi is being use under the hood. This will mean re - writing all these classes and the only reason we have chosen this path was to actually not have to do this. I am going to try to see how I can remove it but I am not putting my hopes here as I do not have any unit test in place to be able to refactor this code.

Reply:
Is it used for some network transport? We have our own anyway, so we should replace it.

Reply:
yup will look into in future tasks once I will have more context here and more safety nets.